### DISCUSSION TITLE
Exercise: Level 3 #3

### DISCUSSION THREAD

**Discussion by leviathan125** [Asabeneh/30-Days-Of-JavaScript#663] [Q&A]
Hello I am not sure how to tackle this question because this day doesn't really cover the regexp. 
This is the code that I have so far and I am only able to clear the % symbol please help:

`const sentence = '%I $am@% a %tea@cher%, &and& I lo%#ve %te@a@ching%;. The@re $is no@th@ing; &as& mo@re rewarding as educa@ting &and& @emp%o@weri@ng peo@ple. ;I found tea@ching m%o@re interesting tha@n any ot#her %jo@bs. %Do@es thi%s mo@tiv#ate yo@u to be a tea@cher!? %Th#is 30#Days&OfJavaScript &is al@so $the $resu@lt of &love& of tea&ching'

let patternFive = /%/g, patternSix = /$/g, patternSeven = /&/g, patternEight = /@/g, patternNine = /;/g
console.log((sentence.replace(patternFive, '')))
`

**Comment by Al-Ponce** [comment]
Hello there, 
I think it doesn't matter if you could solve the quest.
  (and you probably missed the # sign
  
    console.log(sentence.replace(/[%$@&#;]/g, ''))
